live-examples/js-examples/atomics-store.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js" class="language-js">// create a SharedArrayBuffer with a size in bytes
+var buffer = new SharedArrayBuffer(16);
+var uint8 = new Uint8Array(buffer);
+uint8[0] = 5;
+
+console.log(Atomics.store(uint8, 0, 2));
+// expected output: 2
+</code>
```

Comment:
Nit: consider adding a `load` call, to show it's really in there:

```
// create a SharedArrayBuffer with a size in bytes
var buffer = new SharedArrayBuffer(16);
var uint8 = new Uint8Array(buffer);
uint8[0] = 5;

console.log(Atomics.store(uint8, 0, 2));
// expected output: 2

console.log(Atomics.load(uint8, 0));
// expected output: 2
```

Reply:
OK